From e28f1ecf2b59696b2d55f758145ecea8d53302ea Mon Sep 17 00:00:00 2001
From: Thierry FOURNIER <tfournier@exceliance.fr>
Date: Wed, 9 Oct 2013 15:23:01 +0200
Subject: [PATCH 80/85] BUILD/MINOR: missing header file

In the header file "common/regex.h", the C keyword NULL is used. This
keyword is referenced into the header file "stdlib.h", but this is not
included.
---
 include/common/regex.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/common/regex.h b/include/common/regex.h
index 0104019..1cc471b 100644
--- a/include/common/regex.h
+++ b/include/common/regex.h
@@ -22,6 +22,8 @@
 #ifndef _COMMON_REGEX_H
 #define _COMMON_REGEX_H
 
+#include <stdlib.h>
+
 #include <common/config.h>
 
 #ifdef USE_PCRE
-- 
1.8.1.5

